src/core/hle/service/hid/hidbus.cpp

```diff
@@ -0,0 +1,322 @@
+             data.size(), bus_handle_.abstracted_pad_id, bus_handle_.bus_type,
+             bus_handle_.internal_index, bus_handle_.player_number, bus_handle_.is_valid);
+
+    IPC::ResponseBuilder rb{ctx, 2};
+    rb.Push(ResultSuccess);
+};
+
+void HidBus::GetSendCommandAsynceResult(Kernel::HLERequestContext& ctx) {
+    IPC::RequestParser rp{ctx};
+    const auto bus_handle_{rp.PopRaw<HidbusBusHandle>()};
+    const std::vector<u8> data = ringcon.GetReply();
+
+    LOG_INFO(Service_HID,
+             "called  abstracted_pad_id={} bus_type={} internal_index={} "
+             "player_number={} is_valid={}",
+             bus_handle_.abstracted_pad_id, bus_handle_.bus_type, bus_handle_.internal_index,
+             bus_handle_.player_number, bus_handle_.is_valid);
+
+    const u64 data_size = ctx.WriteBuffer(data);
+    IPC::ResponseBuilder rb{ctx, 4};
+    rb.Push(ResultSuccess);
+    rb.Push<u64>(data_size);
+};
+
+void HidBus::SetEventForSendCommandAsycResult(Kernel::HLERequestContext& ctx) {
+    IPC::RequestParser rp{ctx};
+    const auto bus_handle_{rp.PopRaw<HidbusBusHandle>()};
+
+    LOG_INFO(Service_HID,
+             "called  abstracted_pad_id={} bus_type={} internal_index={} "
+             "player_number={} is_valid={}",
+             bus_handle_.abstracted_pad_id, bus_handle_.bus_type, bus_handle_.internal_index,
+             bus_handle_.player_number, bus_handle_.is_valid);
+
+    IPC::ResponseBuilder rb{ctx, 2, 1};
+    rb.Push(ResultSuccess);
+    rb.PushCopyObjects(send_command_asyc_event->GetReadableEvent());
+};
+
+void HidBus::GetSharedMemoryHandle(Kernel::HLERequestContext& ctx) {
+    LOG_DEBUG(Service_HID, "called");
+
+    IPC::ResponseBuilder rb{ctx, 2, 1};
+    rb.Push(ResultSuccess);
+    rb.PushCopyObjects(&system.Kernel().GetHidBusSharedMem());
+}
+
+void HidBus::EnableJoyPollingReceiveMode(Kernel::HLERequestContext& ctx) {
+    IPC::RequestParser rp{ctx};
+    const auto t_mem_size{rp.Pop<u32>()};
+    const auto t_mem_handle{ctx.GetCopyHandle(0)};
+    const auto polling_mode_{rp.PopRaw<JoyPollingMode>()};
+    const auto bus_handle_{rp.PopRaw<HidbusBusHandle>()};
+
+    ASSERT_MSG(t_mem_size == 0x1000, "t_mem_size is not 0x1000 bytes");
+
+    auto t_mem =
+        system.CurrentProcess()->GetHandleTable().GetObject<Kernel::KTransferMemory>(t_mem_handle);
+
+    if (t_mem.IsNull()) {
+        LOG_ERROR(Service_HID, "t_mem is a nullptr for handle=0x{:08X}", t_mem_handle);
+        IPC::ResponseBuilder rb{ctx, 2};
+        rb.Push(ResultUnknown);
+        return;
+    }
+
+    ASSERT_MSG(t_mem->GetSize() == 0x1000, "t_mem has incorrect size");
+
+    ringcon.SetPollingMode(polling_mode_);
+    ringcon.SetTransferMemoryPointer(system.Memory().GetPointer(t_mem->GetSourceAddress()));
+
+    LOG_INFO(Service_HID,
+             "called t_mem_handle=0x{:08X} polling_mode={} abstracted_pad_id={} bus_type={} "
+             "internal_index={} "
+             "player_number={} is_valid={}",
```

Comment:
```suggestion
             "called, t_mem_handle=0x{:08X}, polling_mode={}, abstracted_pad_id={}, bus_type={}, internal_index={}, player_number={}, is_valid={}",
```

Then let clang-format work its magic to format this properly, ditto for all other logging

Reply:
Uh not sure how that happen. Must the same clang that created those new lines.